app/src/main/res/values/strings.xml

```diff
@@ -74,4 +74,7 @@
     <string name="channel_name">Chess Clock</string>
     <string name="channel_description">Chess Clock Main Channel</string>
 
+    <string name="x_min">%d min</string>
+    <string name="x_min_y_sec">%d min | %2d sec</string>
```

Comment:
for i18n freedom it would be better to have those in plurals, but since we are En focused it's probably fine as is.